scripts/publish_packages.sh

```diff
@@ -56,6 +56,10 @@ if [[ "${TRAVIS_PUBLISH_PACKAGES:-}" == "true" ]]; then
         --skip-existing \
         --verbose
 
+    echo "Publishing .nupkgs to nuget.org:"
+    find ~/.nuget/local/ -name '*.nupkg' \
```

Comment:
This seems a bit non-specific... what if other NuGet packages end up in this folder?

Reply:
fair point.  probably best to just enumerate the packages explicitly.  Or at least filter to those that start with Pulumi.

Reply:
have changed this to only upload Pulumi* nupackages.